commons/types/account_test.go

```diff
@@ -77,37 +79,97 @@ func TestAccountCache(t *testing.T) {
 
 //TestToAccountByAddress
 func TestToAccountByAddress(t *testing.T) {
-	// TODO: ToAccountByAddress()
-	t.Skip("Need a Badger DB mock")
+	defer destruct()
+	txn := db.NewTransaction(true)
+	defer txn.Discard()
+	account := &Account{}
+	account.UnmarshalJSON(testAccountByte)
+	account.Persist(txn)
+
+	testAccount,err := ToAccountByAddress(txn, account.Address)
+	if err != nil{
+		t.Error(err)
+	}
+	if reflect.DeepEqual(testAccount, account) == false{
+		t.Error("account not equal to testAccount")
+	}
 }
 
 //TestToAccountByName
 func TestToAccountByName(t *testing.T) {
-	// TODO: ToAccountByName()
-	t.Skip("Need a Badger DB mock")
```

Comment:
Test functions either need to have a body or else have the t.Skip method called.